### DISCUSSION TITLE
Disabling position auto-adjustment in the panel

### DISCUSSION THREAD

**Discussion by tmiroslav** [grafana/grafana#38142] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

- I believe it would be very useful if there is an option to disable position auto-adjustment in the panel, through some config file for example. I have already posted question in Grafana Slack [here](https://grafana.slack.com/archives/C05675Y4F/p1597950678023200), and have not found out that such option exist.

**Why is this needed**:
It's needed because with this auto-adjustment Grafana does not provide ability to provision panel object  (in our case singlestat) in the position where is best from visual perspective to understand where is the problem in the system exactly. Our platform is running in the cloud and have so many services that we monitor, and when problem with service happen it is crucial to understand where it happens, meaning to see immediately which service group it belongs and it which AZ it is running. Auto-adjustment feature in the panel dashboard makes us a lot of trouble to achieve this clarity. 

Thank you for considering this.
Miroslav

**Comment by tmiroslav** [comment]
@peterholmberg I was wondering how I can follow progress on this topic? Thank you!

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
